group_vars/glance.yml

```diff
@@ -2,4 +2,4 @@
 # Global Variables for 'glance' Inventory Group
 
 glance_filesystem_store_datadir: /var/opt/pf9/imagelibrary/data
-glance_public_endpoint: False
+glance_public_endpoint: True
```

Comment:
This might be an acceptable approach but probably better to specify which one should be public.

Reply:
I'll set the default to False (in group_vars/glance.yml) and then override it in the inventory file like this:

hv301 glance_public_endpoint=true

Reply:
I like this. 

There is the issue where someone does not specify a public endpoint and are left with unusable glance (I'm not sure if this would break the rest of the autodeploy pipeline).

I'm not sure if autodeploy should handle that case, or if we want to leave it up to the user to specify one as a public endpoint. But I could see a lot of users authorizing nodes without specifying a public endpoint and then being unable to use glance. It's not trivial because someone could already have specified a public endpoint and are simply authorizing a few new imagelibrary nodes, and we wouldn't want to automatically switch their public endpoint if it wasn't desired.